neurobooth_os/tasks/task.py

```diff
@@ -25,7 +26,22 @@
 from neurobooth_os.log_manager import APP_LOG_NAME
 import neurobooth_os.config as cfg
 
-from enum import Enum
+from pylsl import local_clock
+
+
+class TaskAborted(Exception):
+    """Exception raised when the task is aborted."""
+    pass
+
+
+def check_if_aborted(keys=("q",)) -> None:
+    """
```

Comment:
do we need the default value for keys here? I've become somewhat suspicious of these, thinking they might degrade maintainability as they propagate. You can certainly convince me otherwise, though.

Reply:
This is a fair point, I will remove the defaults and make it a required argument. Maybe the Task class should specify variables like "abort_keys" that we use in such calls.